cpp/include/cudf/functions.h

```diff
@@ -384,62 +384,17 @@ gdf_error gdf_hash_partition(int num_input_cols,
 /** 
  * @Synopsis  Computes the prefix sum of a column
  * 
- * @Param inp Input column for prefix sum with null_count = 0
- * @Param out The output column containing the prefix sum of the input
- * @Param inclusive Flag for applying an inclusive prefix sum
- * 
- * @Returns   GDF_SUCCESS if the operation was successful, otherwise an appropriate
- * error code. If inp->null_count is not set to 0 GDF_VALIDITY_UNSUPPORTED is
- * returned.
- */
-/* ----------------------------------------------------------------------------*/
-gdf_error gdf_prefixsum_generic(gdf_column *inp, gdf_column *out, int inclusive);
-
-/* --------------------------------------------------------------------------*/
-/** 
- * @Synopsis  Computes the prefix sum of a column
- * 
- * @Param inp Input column for prefix sum with null_count = 0
- * @Param out The output column containing the prefix sum of the input
- * @Param inclusive Flag for applying an inclusive prefix sum
- * 
- * @Returns   GDF_SUCCESS if the operation was successful, otherwise an appropriate
- * error code. If inp->null_count is not set to 0 GDF_VALIDITY_UNSUPPORTED is
- * returned.
- */
-/* ----------------------------------------------------------------------------*/
-gdf_error gdf_prefixsum_i8(gdf_column *inp, gdf_column *out, int inclusive);
-
-/* --------------------------------------------------------------------------*/
-/** 
- * @Synopsis  Computes the prefix sum of a column
- * 
- * @Param inp Input column for prefix sum with null_count = 0
- * @Param out The output column containing the prefix sum of the input
- * @Param inclusive Flag for applying an inclusive prefix sum
- * 
- * @Returns   GDF_SUCCESS if the operation was successful, otherwise an appropriate
- * error code. If inp->null_count is not set to 0 GDF_VALIDITY_UNSUPPORTED is
- * returned.
+ * @Param[in] input Input column for prefix sum with null_count = 0
```

Comment:
The `null_count` doesn't need to be zero anymore, yes?

Reply:
Add a detailed description below the `brief` about how null values in the input are handled, i.e., treated as if they were the identity of the scan binary operation being performed.

Reply:
> The `null_count` doesn't need to be zero anymore, yes?

nulls are already supported.

> treated as if they were the identity of the scan binary operation being performed.

How about this? 
I'd also like to note that the output column has same characteristics at nulls with the input column.

"The Null values are skipped for the operation, and the output column has same null values with the input column. "